pkgs/development/libraries/gpgme/default.nix

```diff
@@ -4,7 +4,12 @@
 let inherit (stdenv) lib system; in
 
 stdenv.mkDerivation rec {
-  name = "gpgme-1.9.0";
+  name = "gpgme-${version}";
+  version = "1.9.0";
+
+  passthru = {
+    pythonSourceRoot = "lang/python";
```

Comment:
is this worth it, considering you only use it for one derivation?

Reply:
If gpgme next updates changes the path, it makes more sense to change the path in gpgme derviation then in the related ones imo. No strong opinion though.

Reply:
While looking at clang package for an unrelated problem, I noticed it had a "python" output . Maybe I should do the same here instead ?
```
    outputs = [ "out" "python" ]
      ++ stdenv.lib.optional enableManpages "man";
```

Reply:
Can the Python bindings be build without building gpgme? If so, then a separate derivation. If not, then an output is typically preferred to reduce building time. It is not required however. I'd say leave it as it is.

Anyway, please remove `pythonSourceRoot`.

Reply:
I removed pythonSourceRoot and rebased